src/main/java/core/basesyntax/service/reportwriter/ReportWriterImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.reportwriter;
+
+import core.basesyntax.db.Storage;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReportWriterImpl implements ReportWriter {
+    private static final String HEAD_LINE = "fruit,quantity";
+    private static final String WRITE_TO_FILE_EXCEPTION = "Can't write to file: ";
+
+    @Override
+    public void writeReport(String fileName) {
+        List<String> collect = Storage.getAll().entrySet()
+                .stream()
+                .map(currentKey -> currentKey.getKey().getName() + "," + currentKey.getValue())
+                .collect(Collectors.toList());
+        File fileTo = new File(fileName);
+        try {
+            collect.add(0, HEAD_LINE);
+            Files.write(fileTo.toPath(), collect);
```

Comment:
this method should only write a string to file. Create a separate interface or add to FruitService a method that would generate a string report from data stored in db